### DISCUSSION TITLE
A tool to help developers know if a browser specific feature is available in all chosen browsers or not

### DISCUSSION THREAD

**Discussion by TheAlmightyMight** [browserslist/browserslist#759] [Ideas]
So, imagine you create a browserslist config and have a CLI tool or something similar that you can prompt whether a CSS/JS feature or API is supported in every single browser your config includes or not. Wouldn't that be awesome? Is there a tool like that?

**Comment by rhianvanesch** [comment]
You can use https://github.com/amilajack/eslint-plugin-compat along with ESLint for JS. For CSS you could use https://github.com/anandthakker/doiuse (though the project hasn't been updated in a while) or https://github.com/ismay/stylelint-no-unsupported-browser-features with Stylelint (also based on `doiuse`). All tools use your browserslist data (e.g. in `.browserslistrc`) to detect which browsers you support.

**Reply by TheAlmightyMight** [reply]
That's awesome! Thank you for these tools, I've been searching for them for a long time
